src/scripts/main.js

```diff
@@ -1,5 +1,43 @@
 'use strict';
 
-// const pushNotification = (top, right, title, description, type) => {
-//   // write code here
-// };
+function pushNotification(
+  top,
+  right,
+  title,
+  description,
+  type,
+  timer = 2000
+) {
+  const notification = document.createElement('div');
+
+  notification.innerHTML = `
+    <h2>${title}</h2>
+    <p>${description}</p>
+  `;
+
+  notification.classList.add('notification');
+  notification.classList.add(type);
```

Comment:
На майбутнє, замість дублювання можна через кому передати або зробити ланцюжком..
```suggestion
  notification.classList.add('notification', type);
  or
  notification.classList.add('notification').add(type);
```